app/views/api_keys/_form.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= form_with model: [:profile, @api_key], data: { controller: "exclusive-checkbox gem-scope" } do |f| %>
   <%= label_tag :name, t("api_keys.index.name"), class: "form__label" %>
-  <%= f.text_field :name, class: "form__input", autocomplete: :off %>
+  <%= f.text_field :name, class: "form__input", autocomplete: :off, disabled: f.object.persisted? %>
```

Comment:
Is name "readonly" once created? Is that enforced also on server side somehow?

Reply:
yes, enforced in the controllers and there are tests for that

Reply:
Looking again, it seems covered by controller. Is that enough? What about model validation?